<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customEntry</class>
 <widget class="QWidget" name="customEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="tokenLabel">
     <property name="text">
      <string>Token</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="issuerLabel">
     <property name="text">
      <string>Issuer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
